Server/core/CServerImpl.h

```diff
@@ -93,8 +93,8 @@ class CServerImpl : public CServerInterface
     SString m_strServerPath;
     SString m_strServerModPath;
 
-    bool m_bRequestedQuit;
-    bool m_bRequestedReset;
+    mutable std::atomic<bool> m_bRequestedQuit;
```

Comment:
its `mutable` by default, isn't it?

Reply:
If my notions from concurrency classes and other programming languages translate to this case in C++, atomic is just only useful if several threads need to do several operations on a value that need not have any effect on any other thread until they all finish. So it's only useful for patterns like incrementing counters, for read-update-write operations.

In the code that this PR touches, only the `load` method is used, which is equivalent to a read operation. If `quit` is never changed in a read-update-write fashion somewhere else, `atomic` is indeed useless and introduces unnecessary atomicity. It'd be enough to guarantee _visibility_, i.e. that other threads will indeed see the updated value, and not that they'll use a stale copy that happens to be in the L1 data cache for a CPU core or something.